src/easy-sql-test.js

```diff
@@ -63,6 +70,10 @@ export default class EasySQLTest {
     let result = [];
 
     for (let query of prepQueries) {
+      if (!query || /^\s+$/.test(query)) {
```

Comment:
I think we should just form those objects "as is". Then if one of the queries will fail inside of `_query()` function well maybe then developer will see throw. I just thinking of a very complex case when array of queries is being generated dynamically. It would be a very subtle bug if one of the generated queries is empty but then it would be "skipped" by the `_convertQueriesToTestSteps()`. Also maybe it makes sense to rename argument to `queries` from `prepQueries`